modules/fs2/build.sbt

```diff
@@ -0,0 +1,32 @@
+name := "pureconfig-fs2"
+
+libraryDependencies ++= Seq(
+  "co.fs2" %% "fs2-core" % "0.10.2",
+  "co.fs2" %% "fs2-io" % "0.10.2",
+  Dependencies.scalaMacrosParadise,
+  Dependencies.scalaTest)
+
+// fs2 0.10 isn't published for Scala 2.10
+crossScalaVersions ~= { oldVersions => oldVersions.filterNot(_.startsWith("2.10")) }
+
+pomExtra := {
+  <scm>
+    <url>git@github.com:pureconfig/pureconfig.git</url>
```

Comment:
You shouldn't specify the SCM information here. Doing that clashes with `sbt-git`, which also injects SCM information in the POM file, leading to invalid (duplicate) SCM entries.

Reply:
Fixed, not sure how that slipped in!